upgrade/1.0.11/Stage_0_Prerequisites.md

```diff
@@ -0,0 +1,181 @@
+# Stage 0 - Prerequisites and Preflight Checks
+
+> **NOTE:** CSM-1.0.1 or later is required in order to upgrade to CSM-1.0.11.
+>
+> **NOTE:** Installed CSM versions may be listed from the product catalog using the following command. This will sort a semantic version without a hyphenated suffix after the same semantic version with a hyphenated suffix, e.g. 1.0.11 > 1.0.11-beta.19.
+>
+
+Use the following command can be used to check the CSM version on the system:
+
+```bash
+ncn# kubectl get cm -n services cray-product-catalog -o json | jq -r '.data.csm'
+```
+
+This check will also be conducted in the 'prerequisites.sh' script listed below and will fail if the system is not running CSM-1.0.1 or CSM-1.0.10.
```

Comment:
Use ` rather than '

Reply:
not sure about this comment -- where would use use 'rather than' in the sentence?